uber/site_sections/jobs.py

```diff
@@ -72,6 +74,7 @@ def signups(self, session, location=None, message=''):
             'location':  location,
             'attendees': session.staffers_for_dropdown(),
             'jobs':      [job_dict(job) for job in session.jobs(location)],
+            'jobsb':     session.jobs(location),
```

Comment:
I don't fully understand the purpose of `jobsb` or why we need both `jobs` and `jobsb`. I would guess that the `job_dict()` function is not serializing something we need. Whatever the case, this looks like a code smell, and that's why we held off on merging this PR.